projects/igniteui-angular/src/lib/tabs/tabs.common.ts

```diff
@@ -15,15 +15,22 @@ export abstract class IgxTabsBase {
     onTabItemSelected: EventEmitter<{}>; // TODO: Define event arg interface!
     hasContentTabs: boolean;
     scrollElement(element: any, scrollRight: boolean) {}
+    performSelectionChange(newTab: IgxTabItemBase) {}
+    transformContentAnimation(tab: IgxTabItemBase, duration: number) {}
+    transformIndicatorAnimation(element: HTMLElement) {}
 }
 
 /** @hidden */
-export interface IgxTabItemBase {
+export abstract class IgxTabItemBase {
     nativeTabItem: ElementRef;
-    select(focusDelay?: number);
+    _isSelected = false;
+    tabindex;
+    select(focusDelay?: number): void {}
```

Comment:
Remove `focusDelay` parameter of the `select` method for both tab and group base classes.

Reply:
Addressed.